core/src/random.rs

```diff
@@ -14,16 +14,16 @@ const STATE_WIDTH: usize = Rpo256::STATE_WIDTH;
 const RATE_START: usize = Rpo256::RATE_RANGE.start;
 const RATE_END: usize = Rpo256::RATE_RANGE.end;
 const HALF_RATE_WIDTH: usize = (Rpo256::RATE_RANGE.end - Rpo256::RATE_RANGE.start) / 2;
-const SEED_HEAD: usize = Rpo256::CAPACITY_RANGE.start;
 
 // RPO RANDOM COIN
 // ================================================================================================
 /// A simplified version of the `SPONGE_PRG` reseedable pseudo-random number generator algorithm
 /// described in https://eprint.iacr.org/2011/499.pdf. The simplification is related to
 /// to the following facts:
 /// 1) A call to the reseed method implies one and only one call to the permutation function.
-///  This is possible because in our case we never reseed with more than a rate number of elements.
-/// 2) As a result of the previous point, we dont make use of input buffer to accumulate seed material.
+///  This is possible because in our case we never reseed with more than 4 field elements.
+/// 2) As a result of the previous point, we dont make use of an input buffer to accumulate seed
+///  material.
```

Comment:
minor nit: let's use `1.` and `2.` for numbering instead of `1)` and `2)` as the doc formatter may have trouble with the latter syntax.

Also, I'd add a comment here to explicitly state that this implementation assumes that `draw_integers()` is called immediately after `reseed_with_int()`.